static/js/click_handlers.js

```diff
@@ -427,26 +427,34 @@ export function initialize() {
         mute_topic(e);
     });
 
+    // RECENT TOPICS
+
     $("body").on("keydown", ".on_hover_topic_mute", convert_enter_to_click);
 
-    $("body").on("click", ".on_hover_topic_unmute", (e) => {
+    $("body").on("click", "#recent_topics_table .on_hover_topic_unmute", (e) => {
         e.stopPropagation();
+        recent_topics.focus_clicked_element(e, 2);
         const stream_id = Number.parseInt($(e.currentTarget).attr("data-stream-id"), 10);
         const topic = $(e.currentTarget).attr("data-topic-name");
         muting_ui.unmute_topic(stream_id, topic);
     });
 
     $("body").on("keydown", ".on_hover_topic_unmute", convert_enter_to_click);
 
-    // RECENT TOPICS
+    $("body").on("click", "#recent_topics_table .on_hover_topic_mute", (e) => {
+        e.stopPropagation();
+        recent_topics.focus_clicked_element(e, 2);
+        mute_topic(e);
```

Comment:
I have sorta the same criticism of passing `e` to `mute_topic` as noted preivously.